pkg/southbound/gnmi/conn.go

```diff
@@ -38,12 +44,21 @@ type ConnID string
 type Conn interface {
 	Client
 	ID() ConnID
+	TargetID() topoapi.ID
+	State() connectivity.State
+	Connect()
+	WatchState(ctx context.Context, ch chan<- connectivity.State) error
 }
 
 // conn gNMI Connection
 type conn struct {
 	*client
-	id ConnID
+	clientConn          *grpc.ClientConn
+	id                  ConnID
+	targetID            topoapi.ID
+	connStateWatchers   []chan<- connectivity.State
```

Comment:
I would just call this `stateWatchers`. `conn` is redundant since that’s the name of the type it’s in already.

Reply:
Renamed